### PR TITLE
🌐 Add norwegian translation for web page

### PR DESCRIPTION
**Pull request by brehen**
Went through all the keys in the translation file for Norwegian and translated sentences and phrases into ones commonly used in Norwegian. 

Left some technology related keywords in their original language, as some of them don't make sense to translate into their Norwegian equivalent. We use a lot of English words in our daily conversations, technologists perhaps most of all.

Added Norwegian to the list of languages on the landing page as well, and changed the order of the languages based on the alphabetical order of their locale. Hope that's fine ✌️ 

Also noticed a bug in the "coming soon" tag for each lesson in the Popover in the Navigation bar, but I'll open a new Pull Request for fixing that. 

Edit: PR for badge translation bug (https://github.com/elixirschool/school_house/pull/202)

### CODE REVIEW

File: lib/school_house_web/templates/page/index.html.heex [hunk 1]
```diff
@@ -10,25 +10,26 @@
...
+        <a href="/it" title="Italian">Italiano</a>,
         <a href="/ja" title="Japanese">日本語</a>,
+        <a href="/ko" title="Korean">한국어</a>,
+        <a href="/ms" title="Malay">Bahasa Melayu</a>,
+        <a href="/no" title="Norwegian">Norsk</a>,
         <a href="/pl" title="Polish">Polski</a>,
         <a href="/pt" title="Portuguese">Português</a>,
+        <a href="/vi" title="Vietnamese">Việt ngữ</a>,
+        <a href="/zh-hans" title="Chinese, in simplified script">简体中文</a>, 
+        <a href="/zh-hant" title="Chinese, in traditional script">繁體中文</a>,
```
Comment (kinson) [inline lib/school_house_web/templates/page/index.html.heex:27]: Thanks for the organization here 🧹 🙂 

Just one suggestion: if you are alphabetizing these in English, then `vi`, `zh-hans`, and `zh-hant` should be at the bottom of the list.

**Review by kinson — COMMENTED** [review]
Thanks for the new translations!!

**Comment by kinson** [comment]
I don't think we have anyone working on Norwegian translations right now (@doomspork would know better though), and if you're interested in helping out more with them then we can create an Elixir School team for Norwegian translators and add you to it!

File: lib/school_house_web/templates/page/index.html.heex (hunk 1 above)
Comment (brehen) [reply lib/school_house_web/templates/page/index.html.heex:27]: That is a very good point! Fixed this now 😄

**Comment by brehen** [comment]
I would love to contribute to the Norwegian translations! 

I was considering starting on some of them yesterday, but I was unsure whether or not the lessons will/should be written as livebook .livemd's or not. If yes, then I figured I might be introducing more debt if I wrote the new translated files in the `"olden ways"`. 🤔 

Would be really exciting to explore writing the lessons in .livemd, it's a banger technology! 💥

File: lib/school_house_web/templates/page/index.html.heex (hunk 1 above)
Comment (kinson) [inline lib/school_house_web/templates/page/index.html.heex:27]: Do you mind moving these to the bottom of the list to be in alphabetical order?

File: lib/school_house_web/templates/page/index.html.heex [hunk 2]
```diff
@@ -10,25 +10,26 @@
...
-        <a href="/uk" title="Ukrainian">Українською</a>,
-        <a href="/ko" title="Korean">한국어</a>,
-        <a href="/it" title="Italian">Italiano</a>,
-        <a href="/de" title="German">Deutsch</a>,
-        <a href="/bn" title="Bengali">বাংলা</a>,
+        <a href="/sk" title="Slovak">Slovenčina</a>, 
         <a href="/tr" title="Turkish">Türkçe</a>,
         <a href="/th" title="Thai">ภาษาไทย</a>, <%= gettext("and")%>
-        <a href="/el" title="Modern Greek">Ελληνικά</a>.
+        <a href="/uk" title="Ukrainian">Українською</a>,
```
Comment (kinson) [inline lib/school_house_web/templates/page/index.html.heex:32]:
```suggestion
        <a href="/uk" title="Ukrainian">Українською</a>.
```

This change will likely be unnecessary once the list is alphabetized, and it's an un-important nitpick, but if you don't mind ending the list with a period that would be great 🙂

**Review by kinson — COMMENTED** [review]
Sorry for the delayed review 😅 ! If you don't mind just updating the list I'll approve/merge this pr! Thanks again for the translations ❤️

File: lib/school_house_web/templates/page/index.html.heex (hunk 2 above)
Comment (brehen) [reply lib/school_house_web/templates/page/index.html.heex:30]: Ah! That is a very fair point! 😄 

It should be good now 👍

File: lib/school_house_web/templates/page/index.html.heex (hunk 1 above)
Comment (brehen) [reply lib/school_house_web/templates/page/index.html.heex]: Moved them down now!

**Review by kinson — APPROVED** [review]
💥

**Pull request merged**